packages/j/jos-animation.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "jos-animation",
+  "description": "An animation library package to instantly add beautiful/professional looking animation to your website",
+  "homepage": "jos-animation.vercel.app",
+  "filename": "dist/jos.js",
+  "keywords": [
+    "jos",
+    "on-scroll",
+    "scroll",
+    "animation",
+    "library",
+    "package",
+    "jesvi",
+    "uikit",
+    "javascript",
+    "lightweight",
+    "performance",
+    "event",
+    "css"
+  ],
+  "authors": [
+    {
+      "name": "Jesvi Jonathan",
+      "email": "jesvi22j@gmail.com",
+      "url": "http://jesvi.pythonanywhere.com"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/jesvijonathan/JOS-Animation-Library.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "jos-animation",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": ["**/*"]
```

```suggestion
        "files": [
          "*.@(css|js)"
        ]
```
Please avoid using open globstars, as they can lead to including files not intended for the CDN. I suspect you just need the CSS/JS files directly in the `dist` directory?

(Also, new line for each item in array, allows for cleaner diffs in the future)

---

Gotchya